terraform/prod/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "app" {}
```

Comment:
You may have already answered this, but who sets this configuration?  It would also be nice to have defaults for some of these in the ASG module rather than passing them all.

Reply:
The `terraform.tfvars` and `backend.tfvars` files just live on my computer for now. Another item for the backlog is to figure out config management/secrets storage for some of this. Since this is a public venue, I'm being cautious about what I include here. Note that, at the moment, there's nothing here that can't be easily discovered with access to the AWS console.